nifi-nar-bundles/nifi-jwt-bundle/nifi-jwt-nar/pom.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Was there an intentional decision not to put these processors under `nifi-standard-processors`?

Reply:
No, I thought this was the best place for the processors and where they should go but if they should be somewhere else let me know!